website/source/docs/agent/options.html.md

```diff
@@ -1724,6 +1724,8 @@ port.
 
 * DNS Interface (Default 8600). Used to resolve DNS queries. TCP and UDP.
 
+* gRPC API (Default 8302). Currently gRPC is only used to expose Envoy xDS API to Envoy proxies.
```

Comment:
I think the gRPC port recommends 8502, sort there is sort-of a default, but not truly. I guess given that we should also say that it is only if configured?

Reply:
Yeah I'd suggest something like:

```
 * gRPC API (Optional, Defaults to 8502 in `-dev` mode). Currently gRPC is only used to expose the xDS API to Envoy proxies. It is off by default but port 8502 is a convention used by various tools as the default.
```

Should we also consider adding HTTPS here? It's in the same boat - off by default so there is no actual default but we recommend `8501` for consistency across tools etc.

Reply:
sorry I missed this was merged already